app/md_extensions_strings.grdp

```diff
@@ -173,8 +173,8 @@
   <message name="IDS_MD_EXTENSIONS_ITEM_EXTENSION_WEBSITE" desc="Label for button to go to the website the extension developer specified.">
     Open extension website
   </message>
-  <message name="IDS_MD_EXTENSIONS_ITEM_CHROME_WEB_STORE" desc="Label for button to visit the Brave web store.">
-    View in Brave Web Store
+  <message name="IDS_MD_EXTENSIONS_ITEM_CHROME_WEB_STORE" desc="Label for button to visit the Web Extensions Store.">
+    View in Web Extensions Store
```

Comment:
Was this file modified manually?
AFAIK, this file is auto generated and changes will be gone during chromium upgrade.

Reply:
See https://github.com/brave/brave-browser/wiki/Strings-and-Localization for more information.

Reply:
Updated to replace string IDs on the fly.